src/Umbraco.Web.BackOffice/Controllers/ContentController.cs

```diff
@@ -1797,6 +1797,11 @@ private bool PublishCulture(IContent persistentContent, IEnumerable<ContentVaria
 
     private IEnumerable<string> GetPublishedCulturesFromAncestors(IContent? content)
     {
+        if (content?.ParentId != -1 && content?.HasIdentity == false)
```

Comment:
```suggestion
        if (content?.ParentId is not -1 && content?.HasIdentity is false)
```
Nitpicking but pattern matching is the new hotness 🔥

Reply:
Haha I forgot.. excellent! 👍